spec/services/calculators/motoring/adult/penalty_points_spec.rb

```diff
@@ -6,20 +6,14 @@
   let(:disclosure_check) { build(:disclosure_check,
                                  under_age: under_age,
                                  known_date: known_date,
-                                 motoring_endorsement: motoring_endorsement) }
+                                 motoring_endorsement: GenericYesNo::YES) }
```

Comment:
As we don't use or set this attribute for penalty points anymore, probably best to cleanup this spec from any trace of endorsements to avoid confusion. Sorry I forgot to mention this in my first review!

Also, I realised the spec `motoring/youth/penalty_points_spec.rb` will require the same changes.

Reply:
Good spot! Have made the changes to this file and the spec `motoring/youth/penalty_points_spec.rb`